Hakawai/Mentions/HKWMentionsCreationStateMachineProtocol.h

```diff
@@ -0,0 +1,75 @@
+#import "HKWChooserViewProtocol.h"
+#import "HKWTextView.h"
+#import "HKWTextView+Plugins.h"
+#import "HKWMentionsDefaultChooserViewDelegate.h"
+#import "HKWMentionsPlugin.h"
+
+@protocol HKWMentionsCreationStateMachineProtocol <HKWMentionsDefaultChooserViewDelegate, HKWMentionsCustomChooserViewDelegate>
```

Comment:
i think this can be called HKWMentionsCreationStateMachineDelegate, instead. which would be cleared